metro/proto/v1/spec.proto

```diff
@@ -1038,4 +1050,10 @@ message ListProjectSubscriptionsRequest {
 message ListProjectSubscriptionsResponse {
     // The names of subscriptions created in the specified project.
     repeated string subscriptions = 1;
-}
\ No newline at end of file
+}
+
+message GetSubscriptionRequest {
+    // Required. The subscription from which messages should be pulled.
```

Comment:
this seems off

Reply:
Comment for GetSubscriptionRequest corrected